### ISSUE TITLE
 [Questchaining - Stormheim] - Vydhar (NPCID: 93231)

### ISSUE THREAD

**Issue by Arkanheal** [GuabinaCore/WoWKoi#1896]
**Links:**
https://www.wowhead.com/npc=93231/vydhar
from WoWHead or our Armory

**What is happening:**
1. You can pick up Lay Them to Rest when reaching Vydhar.
2. You can pick up "Bjornhata" after completing "To Haustvald"

**What should happen:**
1. Lay Them to Rest is part of a questchain, you should have to complete quests before beeing able to pick it up:
     1. https://www.wowhead.com/quest=39796/to-haustvald
     2. https://www.wowhead.com/quest=38778/turn-the-keys & https://www.wowhead.com/quest=39788/the-runewoods-revenge
     3. https://www.wowhead.com/quest=38808/bjornharta & https://www.wowhead.com/quest=38810/the-dreaming-fungus

Basically you should be able to pick it up when you can pick [this one](https://www.wowhead.com/quest=38811/judgment-day)

2. Bjornhata should only be pickable after completing those 2 quests:
    1. https://www.wowhead.com/quest=38778/turn-the-keys & https://www.wowhead.com/quest=39788/the-runewoods-revenge

**Comment by Cefiro-Caecias** [comment]
Fixed. Available in the next update.
